test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-3.xml

```diff
@@ -1621,7 +1621,359 @@
+		<command>$JAVA_EXE$ $currentMode$,printstats=all</command>
+		<output type="success" caseSensitive="yes" regex="no">ROMCLASS: java/lang/Object at</output>
+
+		<output type="required" caseSensitive="yes" regex="no">ROMCLASS: org/openj9/test/ivj/Hanoi</output>
+		<output type="required" caseSensitive="yes" regex="no">Current statistics for top layer of cache</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">cache layer[\s]*= 2</output>
+		<output type="required" caseSensitive="yes" regex="no">Current statistics for all layers of cache</output>	
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">ROMClass bytes</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes"># ROMClasses</output>			
+		
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 189: Test -Xshareclasses:printtoplayerstats=all" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printtoplayerstats=all</command>
+		<output type="success" caseSensitive="yes" regex="no">CLASSPATH</output>
+
+		<output type="required" caseSensitive="yes" regex="no">ROMCLASS: org/openj9/test/ivj/Hanoi</output>
+		<output type="required" caseSensitive="yes" regex="no">ROMMETHOD:</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">cache layer[\s]*= 2</output>
+
+		<output type="failure" caseSensitive="yes" regex="no">ROMCLASS: java/lang/Object at</output>
+		<output type="failure" caseSensitive="yes" regex="no">Current statistics for all layers of cache</output>	
+		<output type="failure" caseSensitive="yes" regex="no">ORPHAN:</output>
+		<output type="failure" caseSensitive="yes" regex="no">AOTCH:</output>
+		<output type="failure" caseSensitive="yes" regex="no">AOTDATA:</output>
+		<output type="failure" caseSensitive="yes" regex="no">AOTTHUNK:</output>
+		<output type="failure" caseSensitive="yes" regex="no">JCL:</output>
+		<output type="failure" caseSensitive="yes" regex="no">BYTEDATA:</output>
+		<output type="failure" caseSensitive="yes" regex="no"># Java Objects</output>
+		<output type="failure" caseSensitive="yes" regex="no">Class LineNumberTable bytes</output>
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 190: Test -Xshareclasses:printtoplayerstats=romclass" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printtoplayerstats=romclass</command>
```

Comment:
Instead of checking for "ROMCLASS", can you check romclass: "org/openj9/test/ivj/Disk" ?